salesagility/SuiteCRM
# Fixes #9257 Adjusting references and tests to reflect updated GoogleA…

…PIalias


Google updated references for their Google API Services "time": "2021-08-15T11:18:25+00:00"
This failed one of our tests which then identified to update other references within the code. 

## Note on current 7.11.21 release. This functionality still works as expected and have successfully ran Google Calendar Sync - only the single test fails. This must be due to maintaining aliases...

However, better to update now than later and that the tests accurately reflect the references called.




## Description





## Motivation and Context


## How To Test This

Need to ensure you have installed the latest composer libraries.
Confirm that all existing functionality of SuiteCRM Google Calendar Sync works as expected. Please see https://docs.suitecrm.com/admin/administration-panel/google-sync/ for step by step guide to setting up Google calendar sync.

## Types of changes


### Final checklist
